.binstar.yml

```diff
@@ -0,0 +1,15 @@
+package: nbpresent
+user: ijstokes
```

Comment:
Is it necessary to name a user? Does it just inherit from the local environment if left blank?